README.md

```diff
@@ -28,10 +28,15 @@ Two-factor authentication support for Bitbucket|&#10003;|&#10003;|&#10003;
 Windows Integrated Authentication (NTLM/Kerberos) support|&#10003;|_N/A_|_N/A_
 Basic HTTP authentication support|&#10003;|&#10003;|&#10003;
 Proxy support|&#10003;|&#10003;|&#10003;
+`amd64` support|&#10003;|&#10003;|&#10003;
+`x86` support|&#10003;|_N/A_|&#10003;
```

Comment:
```suggestion
`x86` support|&#10003;|_N/A_|&#10007;
```

We don't have any `x86` builds for Linux, and not sure if we've tested `x86` on Linux.

Also technically we don't have any native `amd64` bits for Windows, but rely on WoW on 64-Bit Windows to run `x86` binaries.